tests/yamitranscode.cpp

```diff
@@ -73,24 +73,14 @@ static void print_help(const char* app)
     printf("   --refmode <VP9 Reference frames mode (default 0 last(previous), "
            "gold/alt (previous key frame) | 1 last (previous) gold (one before "
            "last) alt (one before gold)> optional\n");
-}
-
-static VideoRateControl string_to_rc_mode(char *str)
-{
-    VideoRateControl rcMode;
-
-    if (!strcasecmp (str, "CBR"))
-        rcMode = RATE_CONTROL_CBR;
-    else if (!strcasecmp(str, "VBR")) {
-        rcMode = RATE_CONTROL_VBR;
-    }
-    else if (!strcasecmp (str, "CQP"))
-        rcMode = RATE_CONTROL_CQP;
-    else {
-        printf("Unsupport  RC mode\n");
-        rcMode = RATE_CONTROL_NONE;
-    }
-    return rcMode;
+    printf("   --profile optional;\n"
```

Comment:
@xuguangxin ,
As HEVC specific, the profile name is main, but it conflicts with AVC here.
I am not so familiar with VP9, do you mean I should remove profile setting for VP9?